src/v3/src/components/AuthCoin/AuthCoin.tsx

```diff
@@ -69,10 +68,15 @@ const AuthCoin: FunctionComponent<AuthCoinProps> = (props) => {
     <Box
       className={containerClasses}
       data-se="factor-beacon"
+      sx={(theme) => ({
+        '--PrimaryFill': theme.palette.primary.dark,
+        '--SecondaryFill': theme.palette.primary.light,
+        '--BackgroundFill': theme.palette.grey[50],
+      })}
```

Comment:
nice! I remember struggling to figure out a good way to handle this so we could move this component over to the newer MUI theme. Glad you worked it out.